### PR TITLE
Adds mats/origin tech/burn_state to more items.

### PR DESCRIPTION
**Pull request by harmonyn**
Exactly what the title says. The only one that's a huge deal is the supermatter sword, but if there's a supermatter sword in the round this is hardly going to be the worst balance concern, and R&D deserves to be able to <s>get rid of it forever</s> use it for research.

### CODE REVIEW

File: code/game/objects/items/weapons/melee/misc.dm [hunk 1]
```diff
@@ -138,6 +139,7 @@
 	armour_penetration = 1000
 	var/obj/machinery/power/supermatter_shard/shard
 	var/balanced = 1
+	origin_tech = "combat=5;materials=3"
```
Comment (GunHog) [inline code/game/objects/items/weapons/melee/misc.dm]: It does not really matter, but materials 3 is outright pathetic for something that is called "super"matter, lol

**Comment by WJohn** [comment]
Yeah should probably be materials 7 or 8 or something.

**Comment by bgobandit** [comment]
sure makes sense

**Comment by Incoming5643** [comment]
honestly, the way the supermatter sword works I'm not sure there's a safe way to put it in a deconstructor. It might eat through it.

**Comment by bgobandit** [comment]
yeah I mean like c4 has levels

**Comment by PKPenguin321** [comment]
one problem with this is that any item with mats in it can be destoryed by an autolathe (for instance, you can destroy .357 traitors revolvers by clicking an autolathe with them), and any items with origin tech can be destroyed by DAs. making gangtools so easily destructible is kinda bad, since that means that even if the gang leader later gets cloned, he's cucked out of having a gangtool.

**Comment by Bawhoppen** [comment]
You can always incinerate and space shit

**Comment by bgobandit** [comment]
>even if the gang leader later gets cloned, he's cucked out of having a gangtool.

intentional, the gang leader should git gud
also, it's not like rival gangsters/security can't just hide/space gangtools

**Comment by GunHog** [comment]
If someone's gang tool gets looted, chances are that he is not going to be getting it back.

File: code/game/objects/items/weapons/melee/misc.dm [hunk 2]
```diff
@@ -14,6 +14,7 @@
 	origin_tech = "combat=4"
 	attack_verb = list("flogged", "whipped", "lashed", "disciplined")
 	hitsound = 'sound/weapons/slash.ogg' //pls replace
+	materials = list(MAT_METAL = 1000)
```
Comment (Dorsisdwarf) [inline code/game/objects/items/weapons/melee/misc.dm]: !!!!!!!! destroying the chain of command? HERESY

File: code/game/objects/items/toys.dm [hunk 3]
```diff
@@ -1220,7 +1220,7 @@
 	icon_state = "toy_mouse"
 	w_class = 2.0
 	var/cooldown = 0
-
+	burn_state = 0 // Burnable
```
Comment (duncathan) [inline code/game/objects/items/toys.dm]: Someone needs to make these damn things defines

**Pull request merged**